bdk-ffi/src/bdk.udl

```diff
@@ -630,7 +646,7 @@ interface Transaction {
   [Throws=TransactionError]
   constructor(sequence<u8> transaction_bytes);
 
-  string txid();
+  string compute_txid();
```

Comment:
Interesting note from the underlying on the reason for the name change `Deprecated since 0.31.0: txid has been renamed to compute_txid to note that it’s computationally expensive.  use compute_txid() instead`